### PR TITLE
MOOSE simulator @3.0.2 (new script)

### PR DESCRIPTION
**Pull request by dilawar**

### CODE REVIEW

File: moose.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+class Moose < Formula
+  desc "Multiscale Object Oriented Simulation Environment"
+  homepage "http://moose.ncbs.res.in"
+  url "https://github.com/BhallaLab/moose-core/archive/macos.tar.gz"
```
Comment (dpo) [inline moose.rb]: We need a `sha256`.

File: moose.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  version "3.0.2"
+
+  depends_on "cmake" => :build
+  depends_on "gsl"
+  depends_on "homebrew/science/hdf5"
+  depends_on "homebrew/science/libsbml" => :optional
```
Comment (dpo) [inline moose.rb]: There's no need to fully qualify `homebrew/science` here.

**Comment by dilawar** [comment]
Seems like some github api errors. Build is OK. It passed on 2 output of 3 builds.

**Comment by dilawar** [comment]
Failing with following error:


~~~~
Error: GitHub API rate limit exceeded for 208.78.110.199. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

Try again in 21 minutes 4 seconds, or create a personal access token:

  https://github.com/settings/tokens/new?scopes=&description=Homebrew

and then set the token as: HOMEBREW_GITHUB_API_TOKEN
~~~~

Is it related to ruby I am trying to submit?

**Comment by saketkc** [comment]
@BrewTestBot test this please

**Comment by dilawar** [comment]
@BrewTestBot  I don't think there is any issue with formula now. The build error is due to `Error: GitHub API rate limit exceeded for 208.78.110.199. (But here's the good n` error at Travis.

File: moose.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
+class Moose < Formula
+  desc "multiscale neuronal modelling software"
+  homepage "http://moose.ncbs.res.in"
+  url "https://github.com/BhallaLab/moose-full/archive/ghevar_3.0.2.tar.gz"
+  # version "3.0.2"
```
Comment (dpo) [inline moose.rb]: This is unnecessary.

**Comment by dpo** [comment]
Please squash your commits.

File: moose.rb [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      ENV["SBML_STATIC_HOME"] = "#{buildpath}/_libsbml_static"
+    end
+
+    args << "-DCMAKE_SKIP_RPATH=ON"
+    mkdir "_build" do
+      system "cmake", "..", *args
+      system "make"
+    end
+
+    Dir.chdir("python") do
```
Comment (dpo) [inline moose.rb]: Just `cd "python" do`.

File: moose.rb [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  homepage "http://moose.ncbs.res.in"
+  url "https://github.com/BhallaLab/moose-core/archive/ghevar_3.0.2-beta.2.tar.gz"
+  version "3.0.2"
+  sha256 "e43d2609425e17e7426955ade60b6b81a699bbe1173ad6af10f09be43fd533eb"
+
+  option "with-gui", "Enable gui support"
+  option "with-sbml", "Enable sbml support"
+
+  depends_on "cmake" => :build
+  depends_on "numpy" => :build
```
Comment (dpo) [inline moose.rb]: This seems redundant with line 17.

File: moose.rb [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+  depends_on "gsl"
+  depends_on "hdf5"
+
+  depends_on "python" if MacOS.version <= :snow_leopard
+  depends_on "numpy" => :python
+
+  if build.with?("gui")
+    depends_on "pyqt"
+    if OS.mac? && MacOS.version <= :mountain_lion
```
Comment (dpo) [inline moose.rb]: We no longer support `:mountain_lion`.

**Comment by dilawar** [comment]
@BrewTestBot The build error is due to Error: GitHub API rate limit exceeded for 208.78.110.199...

**Comment by dpo** [comment]
@BrewTestBot test this please

**Pull request closed without merging**